packages/storybook/stories/va-select-uswds.stories.jsx

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable react/prop-types */
+import React, { useEffect, useState } from 'react';
+import { getWebComponentDocs, propStructure, StoryDocs } from './wc-helpers';
+
+const selectDocs = getWebComponentDocs('va-select');
+
+export default {
+  title: 'USWDS/Select USWDS',
+  id: 'uswds/va-select',
+  parameters: {
+    componentSubtitle: 'va-select web component',
+    docs: {
+      page: () => <StoryDocs data={selectDocs} />,
+    },
+  },
+};
+
+const defaultArgs = {
+  'label': 'Branch of Service',
+  'name': 'branch',
+  'value': 'army',
```

Comment:
Now that we have the `- select -` default value in the component, can we update the Storybook example so that it displays by default (instead of "army")?